style.css

```diff
@@ -0,0 +1,332 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+.my-logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.my-menu {
+  position: relative;
+  cursor: pointer;
+  user-select: none;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+p {
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+a {
+  width: 117px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+section {
+  padding: 12px;
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  background-image: url('img/background.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: white;
+  border-bottom-left-radius: 70px;
+}
+
+.socialnetworks {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 20px;
+}
+
+.socialnetworks1 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 25px;
+}
+
+.socialnetworks2 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 200px;
+  font-size: 24px;
+  color: #000;
+}
+
+.socialnetworks3 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 290px;
+  font-size: 24px;
+  color: #000;
+}
+
+button {
+  border: 1px solid purple;
+  border-radius: 5px;
+  padding: 5px 20px;
+  background-color: white;
+  color: purple;
+  cursor: pointer;

```

Comment:
- [ ] Kindly make sure you stick to the original design provided in Figma. Your buttons and stack list need to be similar to the ones in the design. please fix that.
`Hint: Ti get an element applied styles, Kindly click that element and you will see the styles popup on the right side in Figma`

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205618610-504b41db-87a0-40ac-be0e-a8ce144b6d2b.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205618782-b0413baf-f98a-4570-be87-2d71a62f1e09.png)